src/solid/src/App.tsx

```diff
@@ -0,0 +1,236 @@
+import DapDSSelectSolid from './components/select';
+import DapDSStackSolid from './components/stack'
+import DapDSDatePickerSolid from './components/datepicker';
+import DapDSTextareaSolid from './components/textarea';
+import DapDSCheckboxSolid from './components/checkbox';
+import DapDSComboboxSolid from './components/combobox';
+import { createSignal, For } from 'solid-js';
+import DapDSSnackbarSolid from './components/snackbar';
+
+declare global {
+  interface Window {
+    showDapSnackbar?: (message: string, options?: any) => void
+  }
+}
+
+interface FormFields {
+  name: string;
+  prefix: string;
+  email: string;
+  birthDate: string;
+  product: string;
+  subject: string;
+  message: string;
+  consent: boolean;
+}
+
+export type Product = {
+  id: number;
+  title: string;
+  description: string;
+  category: string;
+  price: number;
+}
+
+function App() {
+  let timeOutId = 0;
+  const [products, setProducts] = createSignal<Product[]>([]);
+  const [form, setForm] = createStore<FormFields>({
+    name: "",
+    prefix: "",
+    email: "",
+    birthDate: "",
+    product: "",
+    subject: "",
+    message: "",
+    consent: false,
+  });
+
+  const [errors, setErrors] = createStore({
+    name: "",
+    prefix: "",
+    email: "",
+    birthDate: "",
+    product: "",
+    subject: "",
+    message: "",
+    consent: "",
+  });
+
+  const getProducts = async(filter: string) => {
+    const response = await fetch(`https://dummyjson.com/products/search?q=${filter}`);
+    const json = await response.json();
+    return json.products.filter((item: any) => item.title.toLowerCase().startsWith(filter.toLowerCase()));
+  }
+
+  const onSubmit = (e: Event) => {
+    e.preventDefault();
+    if (window.showDapSnackbar) {
+      window.showDapSnackbar('Gratulálunk! Minden mező helyes!', {
+        duration: 4500,
+        alertType: 'successful',
+        actions: [
+          { href: 'https://sg.hu', text: 'SG' },
+          { href: 'https://index.hu', text: 'Index' },
+        ],
+      })
+    }
+  }
+
+  const getFormFieldValue = (field: keyof FormFields): string | boolean=> form[field];
+  const updateErrorField = (field: keyof FormFields, value: string) => setErrors({ [field]: value });
+
+  const validateRequired = (field: keyof FormFields, message: string): void => {
+    if (!getFormFieldValue(field)) {
+      updateErrorField(field, message)
+    } else {
+      updateErrorField(field, '')
+    }
+  }
+
+  const validateEmailPattern = (field: keyof FormFields, message: string): void => {
+    const fieldValue = getFormFieldValue(field);
+    if (typeof fieldValue === 'string' && !fieldValue.match(/[^@\s]+@[^@\s]+\.[^@\s]+/)) {
+      updateErrorField(field, message);
+    } else {
+      updateErrorField(field, '');
+    }
+  }
+
+  return (
+    <>
+      <div>
+        <DapDSSnackbarSolid></DapDSSnackbarSolid>
+        <form>
+          <DapDSStackSolid>
+            <DapDSInputSolid
```

Comment:
https://custom-elements-everywhere.com/#solid

```suggestion
            <dap-ds-input
              id="name"
              label="Teljes név"
              name="name"
              feedbackType="negative"
              value={form.name}
              feedback={errors?.name?.toString()}
              on:dds-change={(e: any) => {
                setForm({name: e.detail.value});
                validateRequired('name', 'Add meg a teljes neved!');
              }}
            >
```

Reply:
Javítottam.